docs/rules/component-tags-order.md

```diff
@@ -14,29 +14,36 @@ since: v6.1.0
 
 ## :book: Rule Details
 
-This rule warns about the order of the `<script>`, `<template>` & `<style>` tags.
+This rule warns about the order of the `<script>`, `<script setup>`, `<template>` & `<style>` tags.
 
 ## :wrench: Options
 
 ```json
 {
-  "vue/component-tags-order": ["error", {
-    "order": [ [ "script", "template" ], "style" ]
-  }]
+  "vue/component-tags-order": [
+    "error",
+    {
+      "order": [["script", "template"], "style"]
+    }
+  ]
 }
 ```
 
-- `order` (`(string|string[])[]`) ... The order of top-level element names. default `[ [ "script", "template" ], "style" ]`.
+- `order` (`(string|string[])[]`) ... The order of top-level element selectors. default `[ [ "script", "template" ], "style" ]`.
 
 ### `{ "order": [ [ "script", "template" ], "style" ] }` (default)
 
 <eslint-code-block fix :rules="{'vue/component-tags-order': ['error']}">
 
 ```vue
 <!-- ✓ GOOD -->
-<script>/* ... */</script>
+<script>
+/* ... */
+</script>
 <template>...</template>
-<style>/* ... */</style>
+<style>
+/* ... */
+</style>
```

Comment:
Please revert these code style changes in the docs. The more concise form makes it easier to see the effects of this rule.